Dockerfile.template

```diff
@@ -69,4 +69,6 @@ RUN dl_URL="https://github.com/sbt/sbt/releases/download/v${SBT_VERSION}/sbt-${S
 	sudo tar -xzf sbt.tar.gz -C /opt/ && \
 	sudo ln -s /opt/sbt-* /opt/sbt && \
 	sbt about && \
-	rm -rf sbt.tar.gz ~/project/project ~/project/target
+	rm -rf sbt.tar.gz ~/project/project ~/project/target && \
+	# install mill
+	curl -L https://github.com/com-lihaoyi/mill/releases/download/0.11.2/0.11.2-11-32f947 > mill && chmod +x mill
```

Comment:
We should version this with a variable (`MILL_VER` for example) to make it easier to update the version moving forward

Reply:
done, im hoping the digits after the first dash are standard, because I cant actually find that download link anywhere except for those docs.

Reply:
@ryanbourdais Looks good! The numbers look a bit like a short commit hash to me, but let's keep an eye on this moving forward 👀